.github/workflows/e2e.yml

```diff
@@ -15,12 +15,19 @@ jobs:
         deploytool: ['operator', 'helm']
         globalnet: ['', 'globalnet']
         cable_driver: ['libreswan', 'strongswan', 'wireguard']
+        ovn: ['', 'ovn']
         exclude:
           # Our Helm setup doesn’t know how to deploy other cable drivers
           - deploytool: 'helm'
             cable_driver: 'wireguard'
           - deploytool: 'helm'
             cable_driver: 'libreswan'
+          - ovn: 'ovn'
+            deploytool: 'helm'
+          - ovn: 'ovn'
+            globalnet: 'globalnet'
+          - ovn: 'ovn'
+            cable_driver: 'strongswan'
```

Comment:
If https://github.com/submariner-io/submariner/pull/1096 goes in, this will no longer be valid (or if this goes in, 1096 needs to be updated).